include/roaring/art/art.h

```diff
@@ -99,6 +99,15 @@ size_t art_size_in_bytes(const art_t *art);
  */
 void art_printf(const art_t *art);
 
+typedef bool (*art_validate_cb_t)(const art_val_t *val, const char **reason,
```

Comment:
Do we need `context`? We can always add it since it's an internal API.

Reply:
I was trying to keep art/roaring64 separate: we need to validate each container stored in the leaves, and currently, art has no idea that its leaves store containers.

Reply:
Oh. Huh, no I guess we don't need context, again, not sure what I was thinking.